plugin/src/main/java/io/jenkins/plugins/coverage/metrics/steps/CoverageApiUtil.java

```diff
@@ -10,57 +10,64 @@
 import io.jenkins.plugins.coverage.metrics.model.LineCoverageType;
 import io.jenkins.plugins.coverage.metrics.model.ModifiedLinesBlock;
 
-//TODO: rename to util
 /**
  * Server side model that provides data for the details of line coverage results in modified lines.
  */
-public class LineCoverageViewModel {
+public class CoverageApiUtil {
 
-    private LineCoverageViewModel() {
+    private CoverageApiUtil() {
         //this is a utility class
     }
 
     /**
      * Static method to extract modified lines of code and their respective coverage status from passed Node object.
```

Comment:
"Static method" is unnecessary information

Reply:
Removed.